test/Libraries/DynamoPythonTests/PythonEvalHelperTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿using NUnit.Framework;
+using PythonNodeModels;
+
+namespace DynamoPythonTests
+{
+    [TestFixture]
+    class PythonEvalHelperTests
+    {
+        [Test]
+        public void TestHelperInitialState()
+        {
+            Assert.AreEqual(false, PythonEvaluationHelper.lazy.IsValueCreated);
+            Assert.AreEqual(false, PythonEvaluationHelper.IsCPythonEnabled);
+            Assert.AreEqual(false, PythonEvaluationHelper.IsIronPythonEnabled);
```

Comment:
@DynamoDS/dynamo Any idea that I kept getting this error..
![image](https://user-images.githubusercontent.com/3942418/86880164-53c39c80-c0ba-11ea-9c95-ff521583aac4.png)

Reply:
I have seen this error when I was trying to directly assign the PythonVersion Enum values to a field in PythonEdit tests class. I couldn't find the actual reason why it was happening but when I did the same from inside a helper function, it worked. 
https://github.com/DynamoDS/Dynamo/blob/master/test/Libraries/DynamoPythonTests/PythonEditTests.cs#L33
I am curious to know what would be the difference though. Will have to look more into it.
In your case, PythonEvaluationHelper is the issue I think.

Reply:
What if you make the enum part of the `PythonEvaluationHelper` class, does that work?

Reply:
are your tests actually preloading the libraries?